### PR TITLE
add EPS payment method

### PR DESCRIPTION
**Pull request by thorsten-stripe**
#### Changes proposed in this Pull Request:
- add redirect flow based payment method EPS: https://stripe.com/docs/sources/eps
- this is pretty much equivalent to Giropay, but only supports banks in Austria

-------------------
- [x] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [x] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

File: includes/admin/stripe-eps-settings.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+$webhook_url = WC_Stripe_Helper::get_webhook_url();
+
+return apply_filters( 'wc_stripe_eps_settings',
+	array(
+		'geo_target' => array(
+			'description' => __( 'Relevant Payer Geography: Austria', 'woocommerce-gateway-stripe' ),
+			'type'        => 'title',
+		),
+		// 'guide' => array(
```
Comment (roykho) [inline includes/admin/stripe-eps-settings.php]: These commented lines need to be removed.

File: includes/admin/stripe-eps-settings.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		// 	'description' => __( '<a href="https://stripe.com/payments/payment-methods-guide#eps" target="_blank">Payment Method Guide</a>', 'woocommerce-gateway-stripe' ),
+		// 	'type'        => 'title',
+		// ),
+		'activation' => array(
+			'description' => __( 'Must be activated from your Stripe Dashboard Settings <a href="https://dashboard.stripe.com/account/payments/settings" target="_blank">here</a>', 'woocommerce-gateway-stripe' ),
+			'type'   => 'title',
```
Comment (roykho) [inline includes/admin/stripe-eps-settings.php]: Align this with previous assignments.

File: includes/payment-methods/class-wc-gateway-stripe-eps.php [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class that handles EPS payment method.
+ *
+ * @extends WC_Gateway_Stripe
+ *
+ * @since 4.0.0
```
Comment (roykho) [inline includes/payment-methods/class-wc-gateway-stripe-eps.php]: All versions need to change to 4.1.0

**Comment by thorsten-stripe** [comment]
@roykho requested changes made and checks passed. Can you review?

**Pull request merged**